### DISCUSSION TITLE
Support Angular applications

### DISCUSSION THREAD

**Discussion by mitsuru17** [orestbida/cookieconsent#327] [General]
Hi! Thanks a lot for this amazing tool!
The main question: Any plans to bring support for Angular applications?

**Comment by orestbida** [comment]
Hi @mitsuru17,

although this is not an angular component, you can still use it in an angular app. The latest version ([v3.0.0-rc.1](https://github.com/orestbida/cookieconsent/releases/tag/v3.0.0-rc.1)) also adds typescript types, ideal for angular apps. Example set-up:

![Screenshot (387)](https://user-images.githubusercontent.com/34487268/175535906-bbb587f5-f5e3-4175-b691-8fbb219ee570.png)

You can also use v2.x.x, but you have to declare the script in the `angular.json` config. file and access the plugin through the `window` object.

If your question implied to actually make a dedicated Angular component, then my answer is no, sorry.
